hugo/assets/sass/_base.scss

```
@@ -411,3 +411,13 @@ blockquote {
     }
   }
 }
+
+.td-sidebar-nav {
+  height: auto !important;
+  max-height: 100% !important;
+}
+
+.td-sidebar {
+  height: auto !important;
+  max-height: 100% !important;
+}
```

I think this is equivalent
```suggestion
.td-sidebar, .td-sidebar-nav {
  height: auto !important;
  max-height: 100% !important;
}
```